cosmic-applet-network/i18n/nl/cosmic_applet_network.ftl

```diff
@@ -1,18 +1,20 @@
 network = Netwerk
-airplane-mode = Vliegtuig modus
-airplane-mode-on = Vliegtuig modus staat aan
-turn-off-airplane-mode = Zet aan om Wi-Fi, Bluetooth en Telefonie te activeren.
-wifi = Wi-Fi
-ipv4 = IPv4 Adres
-ipv6 = IPv6 Adres
+airplane-mode = Vliegtuigmodus
+airplane-mode-on = Vliegtuigmodus staat aan
+turn-off-airplane-mode = Schakel uit om wifi, bluetooth en mobile data te activeren.
+wifi = Wifi
+ipv4 = IPv4-adres
+ipv6 = IPv6-adres
 mac = MAC
 megabits-per-second = Mbps
 connected = Verbonden
-connecting = Aan het verbinden
```

Comment:
Line 11: Making it consistent with the https://github.com/pop-os/cosmic-settings/pull/845/files.
"Aan het verbinden" is spoken language, not written language.